src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html

```diff
@@ -138,8 +138,8 @@ <h2> Filter Students </h2>
                   [queryParams]="{courseid: course.id}">
             Enroll
           </button>
-          <i class="fas fa-chevron-down pl-3" [hidden]="!course.isChecked"></i>
-          <i class="fas fa-chevron-up pl-3" [hidden]="course.isChecked"></i>
+          <i class="fas fa-chevron-up pl-3" [hidden]="!course.isChecked"></i>
+          <i class="fas fa-chevron-down pl-3" [hidden]="course.isChecked"></i>
```

Comment:
These lines are all that you need to submit.

Reply:
@wkurniawan07 no problem!
I've reverted the commit with the tests in.
Let me know if there's anything else I can do to help get this merged.